service/service.go

```diff
@@ -367,12 +407,18 @@ func (app *Application) execute() {
 	app.logger.Info("Starting shutdown...")
 
 	app.notifyPipelineNotReady()
-	app.shutdownPipelines()
+
+	err = app.shutdownPipelines()
+	if err != nil {
+		return err
```

Comment:
Similar question here: shall we proceed and combine & return all accumulated errors in the end?

Reply:
Fixed.